File: FLUTTER/igest_app/lib/models/principal/var_global.dart
class VarGlobal {
  int idVarGb;
  int idEmpresa;
  String idSistema;
  String nomVarGb;
  String valor;
  String descripcion;

  VarGlobal({
    required this.idVarGb,
    required this.idEmpresa,
    required this.idSistema,
    required this.nomVarGb,
    required this.valor,
    required this.descripcion,
  });

  factory VarGlobal.fromJson(Map<String, dynamic> json) {
    return VarGlobal(
      idVarGb: json['id_var_gb'],
      idEmpresa: json['id_empresa'],
      idSistema: json['id_sistema'],
      nomVarGb: json['nom_var_gb'],
      valor: json['valor'],
      descripcion: json['descripcion'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id_var_gb': idVarGb,
      'id_empresa': idEmpresa,
      'id_sistema': idSistema,
      'nom_var_gb': nomVarGb,
      'valor': valor,
      'descripcion': descripcion,
    };
  }
}

File: SPRING/igest.server.model/src/main/java/com/yupanastudio/igest/server/principal/dml/VarGlobal.java
/*
 * Document   : VarGlobal
 * Author     : Franklin Barzola Olivares <frankoantonio@gmail.com>
 */
package com.yupanastudio.igest.server.principal.dml;

import org.springframework.data.annotation.Id;

public class VarGlobal {

    @Id
    private Integer idVarGb;
    private int idEmpresa;
    private String idSistema;
    private String nomVarGb;
    private String valor;
    private String descripcion;

    public Integer getIdVarGb() {
        return idVarGb;
    }

    public void setIdVarGb(Integer idVarGb) {
        this.idVarGb = idVarGb;
    }

    public int getIdEmpresa() {
        return idEmpresa;
    }

    public void setIdEmpresa(int idEmpresa) {
        this.idEmpresa = idEmpresa;
    }

    public String getIdSistema() {
        return idSistema;
    }

    public void setIdSistema(String idSistema) {
        this.idSistema = idSistema;
    }

    public String getNomVarGb() {
        return nomVarGb;
    }

    public void setNomVarGb(String nomVarGb) {
        this.nomVarGb = nomVarGb;
    }

    public String getValor() {
        return valor;
    }

    public void setValor(String valor) {
        this.valor = valor;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

}
